src/components/snackbar/snackbar.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { useState, forwardRef, useImperativeHandle } from "react"
+import "./snackbar.css"
+
+export const Snackbar = forwardRef((props, ref) => {
+  const [showSnackbar, setShowSnackbar] = useState(false);
+
+  useImperativeHandle(ref, () => ({
+    show() {
+      setShowSnackbar(true);
+      setTimeout(() => {
+        setShowSnackbar(false);
+      }, 2000);
+    },
+  }))
+  
+  return (
+  <div className="snackbar" id={showSnackbar ? "show" : "hide"}>
+    <div className="symbol">
+      <h1>&#x270C;</h1>
+    </div>
+    <div className="message">
+      <p>{props.message}</p>
```

Comment:
If this is your only props and it's the text that's visible when this component is rendered I would use the `children` prop to create a nicer component API. 

So right now where this is used you have to do 

```jsx
<Snackbar message="Some message" />
```

If you use the `children` prop you can do 
```jsx
<Snackbar>
	Some message
</Snackbar>
```
Which reads more like standard HTML so it's a little easier to parse and understand the final output.

Reply:
Where did you see that? It's best to link to documentation rather than screenshotting it so that I can gather full context.

Reply:
Ah, I think I have found [the documentation you're referring to](https://react.dev/reference/react/Children).

I think what you've run into is `Children` the API vs `children` the prop. It's worth looking at that documentation you've found and reading about it's usage. tl;dr It's a set of functions for manipulating `children` the prop, something you almost never need to do.

What I'm referring to is [passing JSX as children](https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children) which is very common.

Reply:
If you read about the alternatives btw it shows you this 😛 

![image](https://github.com/d-elv/pokeApi/assets/9110415/63e69381-bedb-4e8e-b440-d45f5c538186)

Reply:
Haha okay, good! And yes, very fair re: linking stuff. I've put that into the code now.